react-effects/my-app/src/List.tsx

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import { readItems, type Item } from './read';
+
+export function List() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [items, setItems] = useState<Item[]>([]);
+  const [error, setError] = useState<unknown>();
+
+  // Your code here:
+  //  - When the component mounts:
+  //    - Read the items using `readItems` and update state so the list displays
+  //    - Handle errors from `readItems`
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const data = await readItems();
+        setItems(data);
+        setError(null);
+      } catch (error) {
+        console.error('failed to fetch items', error);
```

Comment:
You need to setError here so you can display the message to the user. If you just log it, the user will never learn there was an error.